tls/s2n_server_cert.c

```diff
@@ -25,7 +25,7 @@
 
 int s2n_server_cert_recv(struct s2n_connection *conn)
 {
-    if (conn->actual_protocol_version == S2N_TLS13) {
+    if (conn->actual_protocol_version >= S2N_TLS13) {
```

Comment:
I don't think this is necessary until we know about TLS14...

Reply:
It's not 'necessary' but could potentially reduce future headache. Do you see harm in it?

Reply:
There's going to be some future headache regardless, but it'll be less if we're consistent  now. So far conditionals have been on conn->actual_protocol_version == S2N_TLS13.

Reply:
"So far conditionals have been on conn->actual_protocol_version == S2N_TLS13." I've been making them ">=" and enforcing them that way in PRs. We've run into a BUNCH of bugs in implementing TLS1.3 because previous checks were written as "== TLS12", so instead of inheriting the expected, most recent behavior, upgrading to TLS13 sent us back to the oldest, legacy behavior. If there is someday a TLS1.4, it will build off of TLS1.3, not TLS1.0.